src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -24,9 +26,12 @@ public class TestJobs2dPatterns {
 	 */
 	private static void setupPresetTests(Application application) {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
-				DriverFeature.getDriverManager());
-
+				DriverFeature.getDriverManager()
+		);
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest("Figure Joe 2", actionEvent -> FiguresJoe.figureScript2(
+				DriverFeature.getDriverManager().getCurrentDriver()
+		));
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)